tools/rerun/rerun_st.py

```diff
@@ -0,0 +1,146 @@
+import multiprocessing
+from openpilot.tools.lib.logreader import LogReader
+from openpilot.tools.lib.route import Route
+import subprocess
+import sys
+import os
+import capnp
+
+
+NUM_CPUS = 1 # Number of CPUS to use in pool
+DEMO_ROUTE = "a2a0ccea32023010|2023-07-27--13-01-19"
+WHEEL_URL = "https://build.rerun.io/commit/660463d/wheels"
+RERUN_VERSION = "rerun-cli 0.16.0-alpha.2 [rustc 1.76.0 (07dca489a 2024-02-04), LLVM 17.0.6] x86_64-unknown-linux-gnu main 660463d, built 2024-04-28T12:33:59Z"
+
+
+try:
+    import rerun as rr
+    import rerun.blueprint as rrb
+    # Simple version check
+    result = subprocess.run(["rerun", "--version"], capture_output=True, text=True)
+    if result.returncode == 0:
+        if result.stdout.strip() != RERUN_VERSION.strip():
+          print("YOU NEED TO UNINSTALL RERUN AND THAN START THIS SCRIPT")
+          print("You have "+ result.stdout)
+          print("You need "+ RERUN_VERSION)
+          print("\nHINT: pip uninstall rerun-sdk")
+          exit(0)
+except ImportError:
+    print("Rerun SDK is not installed. Trying to install it...")
+    # subprocess.run([sys.executable, "-m", "pip", "install", "rerun-sdk"])
+    subprocess.run(
+        [sys.executable, "-m", "pip", "install", "--pre", "-f", WHEEL_URL, "--upgrade", "rerun-sdk"],
+        check=True
+    )
+    print("Rerun installed, restarting script")
+    os.execv(sys.executable, [sys.executable] + sys.argv)
```

Comment:
I think it's a better user experience to require a certain `--install` flag, or ask for confirmation before auto-installing programs. We should copy the PlotJuggler user experience we've already established